.editorconfig

```diff
@@ -0,0 +1,15 @@
+root=true
```

Comment:
Used by KtLint for configuration — it's basically the same as what the Android Kotlin style guidelines dictate, **except** that:
 * Max line length is 150, not 100
 * Continuation indent size is 4, not 8 (for XML, JSON and YML it's 2)